<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScorerWidget</class>
 <widget class="QWidget" name="ScorerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScorerWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Parse</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QPushButton" name="parseConstraintPair">
        <property name="text">
         <string>constraint pair</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="parseConstraintGroup">
        <property name="text">
         <string>constraint group</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="parseGlobalSymm">
        <property name="text">
         <string>global reflection symm</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Evaluate Single</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QPushButton" name="evaluateGlobalSymm">
        <property name="text">
         <string>global reflection symm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="evaluateTopology">
        <property name="text">
         <string>Connectivity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="evaluateGroups">
        <property name="text">
         <string>Groups</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Evaluate auto</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="evaluateGlobalSymmAuto">
           <property name="text">
            <string>global reflection symm</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="evaluateTopologyAuto">
           <property name="text">
            <string>Connectivity</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="evaluateGroupsAuto">
           <property name="text">
            <string>Groups</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Options</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCheckBox" name="bUseSourceCenter">
           <property name="toolTip">
            <string>Use source shape's center when global symm</string>
           </property>
           <property name="text">
            <string>Use source's center</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="bUsePart">
           <property name="text">
            <string>Use part for global symm</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="bUseLink">
           <property name="text">
            <string>Use link for connectivity</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
